Prepare for move to Heroku

davehunt opened the pull request:
Once this is merged I can transfer ownership of the Heroku app to the mozillacorporation team. Then we probably just need to add owners of the app so they can configure it and start/stop it as needed.

/cc @rvandermeulen @stuartphilp

whimboo commented on configuration.js:
```diff
@@ -0,0 +1,15 @@
+  channels: envs('IRC_CHANNELS', '#qa'),
+  secure: envs('IRC_SECURE', 'true'),
+  autoRejoin: envs('IRC_AUTO_REJOIN', 'true'),
+  admins: envs('ADMINS', 'ashughes,whimboo'),
```
Please put one of you in here. We cannot maintain it anymore.

davehunt replied:
One of who? I doubt this is the actual configuration that will be used. I can change this but it should probably not use real examples. All I've done is change how this is configured so it can be controlled via Heroku's environment arguments.

davehunt replied:
I've removed actual nicks from the defaults. We'll configure these directly in Heroku.

whimboo replied:
That's great. Thanks!

whimboo commented on configuration.js:
```diff
@@ -0,0 +1,15 @@
+  secure: envs('IRC_SECURE', 'true'),
+  autoRejoin: envs('IRC_AUTO_REJOIN', 'true'),
+  admins: envs('ADMINS', 'ashughes,whimboo'),
+  helpers: envs('HELPERS', 'ashughes'),
```
Same here, maybe a responsible person from SV?

davehunt replied:
I'm open to suggestions, but like I said above, that would be configured in Heroku.

whimboo commented on package.json:
```diff
@@ -1,8 +1,12 @@
-  , "dependencies": {
-      "irc" : "0.3.7"
+    "name": "testdaybot",
+    "version": "0.0.2a",
```
Please bump up with this change.

davehunt replied:
Sure, I can do that, but we don't actually distribute this as a package.

davehunt commented:
> I would have loved more fine-grained commits.

This really isn't that big a change. All I've changed is the way the bot is configured so that it can be controlled via Heroku, added a Procfile for Heroku, and removed node_modules. I suppose removing the node_modules could have been done separately.
